### PR TITLE
[TACHYON-756][WIP] Add logging component for tachyon access logs

### PR DESCRIPTION
**Pull request by ooq**
JIRA: [TACHYON-756](https://tachyon.atlassian.net/browse/TACHYON-756)

We want to have another logger to record tachyon access logs on both workers and master.
The first commit setups up configuration needed for this logger.

### CODE REVIEW

**Comment by apc999** [comment]
Do we now turn on worker access logger by default now?

**Comment by ooq** [comment]
I will make it disabled by default. In the long term I guess we can decide depending on the overhead.

**Comment by apc999** [comment]
@ooq  having new flags turned off by default is good :)

File: deploy/docker/tachyon-dev/tachyon-base/files/log4j.properties [hunk 1]
```diff
@@ -19,6 +19,17 @@ log4j.appender.MASTER_LOGGER.layout=org.apache.log4j.PatternLayout
 log4j.appender.MASTER_LOGGER.layout.ConversionPattern=%d{ISO8601} %-5p %c{2} (%F:%M) - %m%n
 #log4j.appender.MASTER_LOGGER.layout.ConversionPattern=%d{ISO8601} %-5p %c{2} (%F:%M(%L)) - %m%n
 
+# Appender for Master Access Log
```
Comment (gpang) [inline deploy/docker/tachyon-dev/tachyon-base/files/log4j.properties]: It seems like the docker logging config is slightly different from the project logging config. Do you know why that is?

File: conf/log4j.properties [hunk 2]
```diff
@@ -26,6 +35,15 @@ log4j.appender.WORKER_LOGGER.layout=org.apache.log4j.PatternLayout
 log4j.appender.WORKER_LOGGER.layout.ConversionPattern=%d{ISO8601} %-5p %c{2} (%F:%M) - %m%n
 #log4j.appender.WORKER_LOGGER.layout.ConversionPattern=%d{ISO8601} %-5p %c{2} (%F:%M(%L)) - %m%n
 
+# Appender for Worker Access Log
```
Comment (gpang) [inline conf/log4j.properties]: I'm curious. Do you foresee the access logs for the worker and master to be different formatting?

**Comment by gpang** [comment]
I left a few questions. Thanks!

File: conf/log4j.properties (hunk 2 above)
Comment (ooq) [inline conf/log4j.properties]: Formatting is likely to be same.

File: deploy/docker/tachyon-dev/tachyon-base/files/log4j.properties (hunk 1 above)
Comment (ooq) [inline deploy/docker/tachyon-dev/tachyon-base/files/log4j.properties]: For docker deployment, we are keeping logs in `${tachyon.home}/logs` and for project logging, we are keeping to `${tachyon.logs.dir}`. I don't think we have to make them different, but whoever writes these docker deployment scripts might have better reasons.

**Comment by gpang** [comment]
LGTM

**Comment by uronce-cc** [comment]
+1 LGTM

**Pull request merged**